src/common/JSONManager.py

```diff
@@ -0,0 +1,71 @@
+import json
+from re import A
+
+class JSONManager:
+
+    def __init__(self, filename: str, autosave: bool = True, structure_mod_allowed: bool = False) -> None:
+
+        with open(filename, "r") as fp:
+            self._d = json.load(fp)
+
+        self._autosave = autosave
+        self._structure_mod_allowed = structure_mod_allowed
+        self._filename = filename
+
+    def get(self, path: str):
+        """
+        Path between keys of the dictionary.
+        Must start with a /
+        """
+
+        if len(path) == 0 or path[0] != '/':
+            raise ValueError("Path must begin with a /")
+
+        if path[-1] != "/":
+            path = str(path + "/")
+
+        return JSONManager._intget(path, self._d)
+
+    def set(self, path: str, value) -> None:
+        """
+        Path between keys of the dictionary.
+        Must start with a /
+        """
+
+        if len(path) == 0 or path[0] != '/':
+            raise ValueError("Path must begin with a /")
+
+        if path[-1] != "/":
```

Comment:
Is it correct to put path[-1]? If yes, why did you put it?

Reply:
> Is it correct to put path[-1]? If yes, why did you put it?

In python [-1] is the last element of an array. I need to check if th epath ends with a /, if not I add it

Reply:
Ok clear, thank you!